Demo/Demo/SwiftUIComponents/PayPalWebPayments/PayPalWebViewModel.swift

```diff
@@ -105,6 +103,20 @@ class PayPalWebViewModel: ObservableObject, PayPalWebCheckoutDelegate {
         }
     }
 
+    func resetState() {
+        updateState(.idle)
+        order = nil
+        checkoutResult = nil
+        orderID = nil
+    }
+
+    private func updateOrder(_ order: Order) {
+        DispatchQueue.main.async {
+            self.orderID = order.id
```

Comment:
🧶 : Do we need the ID property since we have `order` as the source of truth?

Reply:
We don't necessarily but it seemed easier to extract this into a constant vs doing `if let orderID = order.id` when we needed to access it in the `paymentButtonTapped` and `createTransaction` method. If we don't find it helpful we can extract as needed.

Reply:
I think we could convert `orderId` into a computed property

```
var orderId: String? {
  order?.id
}
```

Reply:
I dig it, done: https://github.com/paypal/paypal-ios/pull/227/commits/2b0c8db4aa1d593beb2afaffdeb2abbc57620d3c